packages/react/src/Stack/Stack.tsx

```diff
@@ -4,15 +4,16 @@ import type {ResponsiveValue} from '../hooks/useResponsiveValue'
 import {getResponsiveAttributes} from '../internal/utils/getResponsiveAttributes'
 
 const StyledStack = styled.div`
-  --Stack-gap-whenRegular: var(--stack-gap-normal, 16px);
-  --Stack-gap-whenNarrow: var(--stack-gap-normal, 16px);
-  --Stack-gap-whenWide: var(--Stack-gap-whenRegular);
+  --stack-gap-none: 0;
+  --stack-gap-condensed: 0.5rem;
+  --stack-gap-normal: 1rem;
+  --stack-gap-spacious: 2rem;
```

Comment:
The only thing that feels confusing to me is that we're resetting these values that already exist globally 🤔 what if we update those values in primer/primitives at some point?

Reply:
Whoops 🤦‍♂️ I completely forgot about that, let me just use these directly and I'll add in the fallback each time I'm using it 👍